packages/run-protocol/src/runStake/runStakeKit.js

```diff
@@ -128,6 +136,292 @@ const initState = (zcf, startSeat, manager) => {
     updater,
   };
 };
+const helperBehavior = {
+  /**
+   * @param {MethodContext} context
+   * @param {ZCFSeat} seat
+   */
+  getCollateralAllocated: ({ state }, seat) =>
+    seat.getAmountAllocated(KW.Attestation, state.collateralBrand),
+  /**
+   * @param {MethodContext} context
+   * @param {ZCFSeat} seat
+   */
+  getRunAllocated: ({ state }, seat) =>
+    seat.getAmountAllocated(KW.Debt, state.debtBrand),
+  /**
+   * @param {MethodContext} context
+   */
+  getCollateralAmount: ({ state, facets }) => {
```

Comment:
What about:

```suggestion
  getCollateralAmount: ({ state, { helper }) => {
```

so we can leave `helper.getCollateralAllocated(...)` alone?

Reply:
or `const { helper } = facets`?

Reply:
I went with the latter. The double destructuring leads causes a circular type definition.